components/event/EventCard.js

```diff
@@ -102,7 +102,7 @@ export default function EventCard({ manage, event, usernames }) {
                   </span>
                 </div>
                 {event.price?.startingFrom > 0 && (
-                  <div>${event.price?.startingFrom}</div>
+                  <div className="ml-auto mr-[-35px]" >${event.price?.startingFrom}</div>
```

Comment:
Can this be done with flex? px is very specific and fragile

Reply:
i have added the mr-[-35px] to make the price text align on the bottom of the profile picture.

Reply:
Using flex would be better, being that specific will break when something else changes

Reply:
> i have added the mr-[-35px] to make the price text align on the bottom of the profile picture.

I think align-self will work better in this context